docs/source/tutorial/word2vec.rst

```diff
@@ -0,0 +1,370 @@
+Word2vec Implementation with Chainer
```

Comment:
I think "with Chainer" is not needed in the title. Because that is obvious. How about "Word2Vec: Obtain a word embeddings"?